models/shopify.yml

```diff
@@ -32,6 +32,37 @@ models:
       - name: total_price_lifetime
         description: Total amount (in currency) up until and including this `date_month`.
 
+  - name: shopify__customer_email_cohorts
+    description: Each record represents a customer's (on the EMAIL grain) performance in a calendar month.
+    columns:
+      - name: cohort_month
+        description: The month the cohort belongs to, i.e the first month the customer had an order.
+      - name: cohort_month_number
+        description: The 'number' of the `date_month` of the record, i.e. how many months from their start month this cohort occurred
+      - name: customer_cohort_id
+        description: Unique key representing a customer in a given month.
```

Comment:
Would probably mention the fields used to create the `cohort_id` (also it doesn't look like `source_relation` is mentioned here even though it's brought into the final model.